### ISSUE TITLE
breaks on dragonfly 1.1.1

### ISSUE THREAD

**Issue by tomasc** [tomasc/dragonfly-mongoid_data_store#1]
```
NoMethodError (undefined method `marshal_b64_encode' for #<Dragonfly::MongoidDataStore:0x007ffdb6c7a360>)
```

**Comment by asgerb** [comment]
Hey @tomasc, I have a PR for this if you add me to the repo 😉

**Comment by tomasc** [comment]
@asgerb yeeee! will do!

**Comment by tomasc** [comment]
@asgerb actually you can make PR from your fork – do you want to try?

**Comment by asgerb** [comment]
Sure! I've already made the changes in the original repo, but can make a fork and copy them over

**Comment by tomasc** [comment]
I can also give you access, whatever you prefer

**Comment by tomasc** [comment]
fixed by #2
